docs/source/workflows/evaluate.md

```diff
@@ -93,9 +93,9 @@ Running `nat eval` produces a set of artifacts in the configured output director
 ### Evaluator outputs (only when configured)
 Each evaluator produces a separate output file only when that evaluator is explicitly configured in `eval.evaluators`:
 - `trajectory_accuracy_output.json`: Scores and reasoning from the trajectory evaluator for each dataset entry, plus an average score.
-- `rag_accuracy_output.json`: RAGAS AnswerAccuracy scores and reasoning per entry, plus an average score.
-- `rag_groundedness_output.json`: RAGAS ResponseGroundedness scores and reasoning per entry, plus an average score.
-- `rag_relevance_output.json`: RAGAS ContextRelevance scores and reasoning per entry, plus an average score.
+- `accuracy_output.json`: Ragas AnswerAccuracy scores and reasoning per entry, plus an average score.
+- `groundedness_output.json`: Ragas ResponseGroundedness scores and reasoning per entry, plus an average score.
+- `relevance_output.json`: Ragas ContextRelevance scores and reasoning per entry, plus an average score.
```

Comment:
Question: Are these filenames actually updated or was this a find/replace issue?